buildpipeline/DotNet-CoreFx-Trusted-Windows.json

```diff
@@ -176,10 +176,10 @@
         "definitionType": "task"
       },
       "inputs": {
-        "solution": "$(Build.SourcesDirectory)\\corefx\\src\\tests.builds",
+        "solution": "$(Build.SourcesDirectory)\\corefx\\src\\upload-tests.proj",
         "platform": "",
         "configuration": "",
-        "msbuildArguments": "$(PB_CreateHelixArguments) /p:\"Branch=$(SourceBranch)\" /p:\"CloudDropAccountName=$(PB_CloudDropAccountName)\" /p:\"CloudResultsAccountName=$(PB_CloudResultsAccountName)\" /p:\"CloudDropAccessToken=$(CloudDropAccessToken)\" /p:\"CloudResultsAccessToken=$(OutputCloudResultsAccessToken)\" /p:\"HelixApiAccessKey=$(HelixApiAccessKey)\" /p:\"HelixApiEndpoint=$(PB_HelixApiEndPoint)\" /p:\"OfficialBuildId=$(OfficialBuildId)\"",
```

Comment:
It's a command-line variable either way, just for random (point-in-time/different people / mystery) reasons, it's part of PB_CreateHelixArguments for Windows only, but PB_TargetQueue for the other two types of build.

I strongly favor getting all the builds to use the same parameter list if at all possible and the same variable name to compose them, but that's beyond the scope of what I'm trying to get done today.